addons/binding/org.openhab.binding.homematic/src/main/java/org/openhab/binding/homematic/internal/communicator/AbstractHomematicGateway.java

```diff
@@ -173,11 +175,26 @@ public void initialize() throws IOException {
         logger.debug("Used Homematic transfer modes: {}", sb.toString());
         startClients();
         startServers();
+
+        if (!config.getGatewayInfo().isHomegear()) {
+            // delay the newDevice event handling at startup, reduces some API calls
+            long delay = config.getGatewayInfo().isCCU1() ? 10 : 3;
+            ScheduledExecutorService scheduler = ThreadPoolManager.getScheduledPool(GATEWAY_POOL_NAME);
```

Comment:
Is there a specific reason you did not use one of the existing pools instead of creating a new one?